test/pipeline_integration_test.exs

```diff
@@ -7,139 +7,95 @@ defmodule Membrane.VideoCompositor.PipelineTest do
   alias Membrane.Testing.Pipeline, as: TestingPipeline
   alias Membrane.VideoCompositor.Test.Utility, as: TestingUtility
 
-  describe "Checks h264 nx pipeline on " do
-    @describetag :tmp_dir
-
-    test "2s 720p 30fps video", %{tmp_dir: tmp_dir} do
-      video_caps = %RawVideo{
-        width: 1280,
-        height: 720,
-        framerate: {30, 1},
-        pixel_format: :I420,
-        aligned: nil
-      }
-
-      implementation = :nx
-      duration = 2
-      test_h264_pipeline(video_caps, duration, implementation, "short_videos", tmp_dir)
+  @implementations [:ffmpeg, :nx]
+
+  Enum.map(@implementations, fn implementation ->
+    describe "Checks h264 #{implementation} pipeline on" do
+      @describetag :tmp_dir
+
+      test "2s 720p 30fps video", %{tmp_dir: tmp_dir} do
+        video_caps = %RawVideo{
+          width: 1280,
+          height: 720,
+          framerate: {30, 1},
+          pixel_format: :I420,
+          aligned: nil
+        }
+
+        duration = 2
```

Comment:
Put caps in module attributes, you can also just pass duration without assigning it to the variable

Reply:
Bump